source/common/runtime/runtime_features.cc

```diff
@@ -86,6 +86,7 @@ RUNTIME_GUARD(envoy_reloadable_features_quic_support_certificate_compression);
 RUNTIME_GUARD(envoy_reloadable_features_quic_upstream_reads_fixed_number_packets);
 RUNTIME_GUARD(envoy_reloadable_features_quic_upstream_socket_use_address_cache_for_read);
 RUNTIME_GUARD(envoy_reloadable_features_reject_invalid_yaml);
+RUNTIME_GUARD(envoy_reloadable_features_report_load_with_rq_issued);
```

Comment:
Can you add release note about this runtime guard here, please https://github.com/envoyproxy/envoy/blob/main/changelogs/current.yaml

Reply:
ahh, done. 
Thanks.